README.md

```diff
@@ -4,19 +4,19 @@
 CookiesOK is a browser extension which aims to automatically click the accept button on Cookie Consent Notices.
 
 ## Installation
-CookiesOK is available from the Chrome web store [here] (https://chrome.google.com/webstore/detail/cookiesok/afmkbjoakcacgljcdccofbffloabfbni), Opera add-ons [here] (https://addons.opera.com/en/extensions/details/cookiesok/) and Mozilla Add-ons [here] (https://addons.mozilla.org/nl/firefox/addon/cookiesok/)
+CookiesOK is available from the Chrome web store [here](https://chrome.google.com/webstore/detail/cookiesok/afmkbjoakcacgljcdccofbffloabfbni), Opera add-ons [here](https://addons.opera.com/en/extensions/details/cookiesok/) and Mozilla Add-ons [here](https://addons.mozilla.org/nl/firefox/addon/cookiesok/)
```

Comment:
This URL is broken and the extension is not available on the Chrome webstore.

Couldn't find another place to report it, so I apologize for the semi-relevant comment.